Gemfile

```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"                                                                                                                                                                                                                                                          
+
+gem 'cocoapods', '0.39.0'
```

Comment:
ok, i'll change line 1 to use literal strings then.  No reason to use interpolated strings here